### PR TITLE
chore(react-avatar): Reduce bundle size by refactoring styles

### PR DESCRIPTION
**Pull request by behowell**
## Related Issue(s)

- Fixes #

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

| Package & Exports                                                                                            |     Baseline (minified/GZIP) |                           PR |                                                                                                                                                                                                    Change |
| :----------------------------------------------------------------------------------------------------------- | ---------------------------: | ---------------------------: | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
| <samp>react-alert</samp> <br /> <abbr title='bundle-size/Alert.fixture.js'>Alert</abbr>                      | `95.859 kB`<br />`23.096 kB` | `96.053 kB`<br />`22.888 kB` | `194 B` <br />`-208 B`  |
| <samp>react-avatar</samp> <br /> <abbr title='bundle-size/Avatar.fixture.js'>Avatar</abbr>                   | `59.524 kB`<br />`15.558 kB` | `59.718 kB`<br />`15.356 kB` | `194 B` <br />`-202 B`  |
| <samp>react-avatar</samp> <br /> <abbr title='bundle-size/AvatarGroup.fixture.js'>AvatarGroup</abbr>         |   `18.137 kB`<br />`6.67 kB` |   `18.79 kB`<br />`6.868 kB` |  `653 B` <br />`198 B`  |
| <samp>react-avatar</samp> <br /> <abbr title='bundle-size/AvatarGroupItem.fixture.js'>AvatarGroupItem</abbr> | `75.785 kB`<br />`20.059 kB` |  `75.979 kB`<br />`19.86 kB` | `194 B` <br />`-199 B`  |
| <samp>react-persona</samp> <br /> <abbr title='bundle-size/Persona.fixture.js'>Persona</abbr>                |  `66.55 kB`<br />`17.502 kB` | `66.744 kB`<br />`17.291 kB` | `194 B` <br />`-211 B`  |

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                                                             |          Size (minified/GZIP) |
| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------: |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ButtonProviderAndTheme.fixture.js'>react-components: Button, FluentProvider & webLightTheme</abbr>              |  `68.075 kB`<br />`18.416 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> | `207.606 kB`<br />`57.581 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr>                            |  `38.778 kB`<br />`12.337 kB` |
| <samp>react-portal-compat</samp> <br /> <abbr title='bundle-size/PortalCompat.fixture.js'>PortalCompatProvider</abbr>                                                         |    `6.446 kB`<br />`2.185 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/DataGrid.fixture.js'>DataGrid</abbr>                                                                                 | `150.938 kB`<br />`41.062 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/PrimitivesOnly.fixture.js'>Table (Primitives only)</abbr>                                                            |  `47.196 kB`<br />`12.732 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/TableAsDataGrid.fixture.js'>Table as DataGrid</abbr>                                                                 | `133.532 kB`<br />`33.674 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/TableSelectionOnly.fixture.js'>Table (Selection only)</abbr>                                                         |  `81.366 kB`<br />`19.658 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/TableSortOnly.fixture.js'>Table (Sort only)</abbr>                                                                   |  `80.696 kB`<br />`19.466 kB` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/d2f17a39e1ccd8b106a442a77a4a143432cb9d54'>d2f17a39e1ccd8b106a442a77a4a143432cb9d54</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)

  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>InfoButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27572/perf-test-react-components/InfoButton-mount_base.html">16</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27572/perf-test-react-components/InfoButton-mount.html">21</a></td>
            <td>5000</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27572/perf-test-react-components/InfoButton-mount.regression.txt">Possible regression</a></td>
           </tr></table><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27572/perf-test-react-components/Avatar-mount_base.html">undefined</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27572/perf-test-react-components/Avatar-mount.err.txt">err</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27572/perf-test-react-components/Button-mount_base.html">394</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27572/perf-test-react-components/Button-mount.html">381</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Field</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27572/perf-test-react-components/Field-mount_base.html">1209</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27572/perf-test-react-components/Field-mount.html">1236</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27572/perf-test-react-components/FluentProvider-mount_base.html">762</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27572/perf-test-react-components/FluentProvider-mount.html">767</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27572/perf-test-react-components/FluentProviderWithTheme-mount_base.html">101</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27572/perf-test-react-components/FluentProviderWithTheme-mount.html">110</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27572/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">94</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27572/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">93</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27572/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">103</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27572/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">108</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>InfoButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27572/perf-test-react-components/InfoButton-mount_base.html">16</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27572/perf-test-react-components/InfoButton-mount.html">21</a></td>
            <td>5000</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27572/perf-test-react-components/InfoButton-mount.regression.txt">Possible regression</a></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27572/perf-test-react-components/MakeStyles-mount_base.html">1176</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27572/perf-test-react-components/MakeStyles-mount.html">1185</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27572/perf-test-react-components/Persona-mount_base.html">1902</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27572/perf-test-react-components/Persona-mount.html">1853</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27572/perf-test-react-components/SpinButton-mount_base.html">1473</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27572/perf-test-react-components/SpinButton-mount.html">1478</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

File: apps/vr-tests-react-components/src/stories/Avatar.stories.tsx [hunk 1]
```diff
@@ -2,6 +2,7 @@ import { storiesOf } from '@storybook/react';
 import * as React from 'react';
```
Comment (fabricteam) [inline apps/vr-tests-react-components/src/stories/Avatar.stories.tsx:2]: ## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 74 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27572&commitId=582a326962e47176ea0f58b940d6149cee176cf8&env=prod)

✅ There was 2 screenshots added, 0 screenshots removed, 1859 screenshots unchanged, 0 screenshots with different dimensions and 72 screenshots with visible difference.


  <details><summary><b>unknown</b> 74 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.basic - Dark Mode.normal.chromium.png|9| Changed|
|Avatar Converged.basic - High Contrast.normal.chromium.png|13| Changed|
|Avatar Converged.basic - RTL.normal.chromium.png|14| Changed|
|Avatar Converged.basic.normal.chromium.png|14| Changed|
|Avatar Converged.customSize+icon+active.normal.chromium.png|700| Changed|
|Avatar Converged.customSize+name+badge.normal.chromium.png|10| Changed|
|Avatar Converged.size+image+badge.normal.chromium.png|182| Changed|
|Avatar Converged.size+inactive+badge.normal.chromium.png|119| Changed|
|AvatarGroup Converged.layoutStack - High Contrast.default.chromium.png|7| Changed|
|Persona Converged.basic - High Contrast.normal.chromium.png|6| Changed|
|Table layout flex - cell actions.always visible - Dark Mode.hover row.chromium.png|1| Changed|
|Table layout flex - cell actions.always visible - High Contrast.chromium.png|25| Changed|
|Table layout flex - cell actions.always visible - High Contrast.hover row.chromium.png|51| Changed|
|Table layout flex - cell actions.default - Dark Mode.hover row.chromium.png|1| Changed|
|Table layout flex - cell actions.default - High Contrast.chromium.png|25| Changed|
|Table layout flex - cell actions.default - High Contrast.hover row.chromium.png|51| Changed|
|Table layout flex - headers.sortable - High Contrast.chromium.png|25| Changed|
|Table layout flex - headers.sortable - High Contrast.hover header.chromium.png|25| Changed|
|Table layout flex - headers.sortable - High Contrast.press header.chromium.png|25| Changed|
|Table layout flex.multiselect (checked) - brand - Dark Mode.chromium.png|3| Changed|
|Table layout flex.multiselect (checked) - brand - High Contrast.chromium.png|201| Changed|
|Table layout flex.multiselect (checked) - brand - RTL.chromium.png|3| Changed|
|Table layout flex.multiselect (checked) - brand.chromium.png|3| Changed|
|Table layout flex.multiselect (checked) - neutral - High Contrast.chromium.png|128| Changed|
|Table layout flex.multiselect (checked) - neutral - RTL.chromium.png|7| Changed|
|Table layout flex.multiselect (checked) - neutral.chromium.png|7| Changed|
|Table layout flex.multiselect (mixed) - brand - Dark Mode.chromium.png|2| Changed|
|Table layout flex.multiselect (mixed) - brand - High Contrast.chromium.png|113| Changed|
|Table layout flex.multiselect (mixed) - brand - RTL.chromium.png|1| Changed|
|Table layout flex.multiselect (mixed) - brand.chromium.png|1| Changed|
|Table layout flex.multiselect (mixed) - neutral - High Contrast.chromium.png|77| Changed|
|Table layout flex.multiselect (mixed) - neutral - RTL.chromium.png|4| Changed|
|Table layout flex.multiselect (mixed) - neutral.chromium.png|4| Changed|
|Table layout flex.multiselect - High Contrast.chromium.png|25| Changed|
|Table layout flex.single select (checked) - High Contrast.chromium.png|25| Changed|
|Table layout flex.single select (checked) - brand - High Contrast.chromium.png|69| Changed|
|Table layout flex.single select (checked) - neutral - High Contrast.chromium.png|51| Changed|
|Table layout flex.single select (checked) - neutral - RTL.chromium.png|2| Changed|
|Table layout flex.single select (checked) - neutral.chromium.png|2| Changed|
|Table layout flex.single select - High Contrast.chromium.png|25| Changed|
|Table layout flex.size - medium - High Contrast.chromium.png|25| Changed|
|Table layout table - cell actions.always visible - Dark Mode.hover row.chromium.png|1| Changed|
|Table layout table - cell actions.always visible - High Contrast.chromium.png|25| Changed|
|Table layout table - cell actions.always visible - High Contrast.hover row.chromium.png|51| Changed|
|Table layout table - cell actions.default - Dark Mode.hover row.chromium.png|1| Changed|
|Table layout table - cell actions.default - High Contrast.chromium.png|25| Changed|
|Table layout table - cell actions.default - High Contrast.hover row.chromium.png|51| Changed|
|Table layout table - headers.sortable - High Contrast.chromium.png|25| Changed|
|Table layout table - headers.sortable - High Contrast.hover header.chromium.png|25| Changed|
|Table layout table - headers.sortable - High Contrast.press header.chromium.png|25| Changed|
|Table layout table.multiselect (checked) - brand - Dark Mode.chromium.png|3| Changed|
|Table layout table.multiselect (checked) - brand - High Contrast.chromium.png|201| Changed|
|Table layout table.multiselect (checked) - brand - RTL.chromium.png|3| Changed|
|Table layout table.multiselect (checked) - brand.chromium.png|3| Changed|
|Table layout table.multiselect (checked) - neutral - High Contrast.chromium.png|128| Changed|
|Table layout table.multiselect (checked) - neutral - RTL.chromium.png|7| Changed|
|Table layout table.multiselect (checked) - neutral.chromium.png|7| Changed|
|Table layout table.multiselect (mixed) - brand - Dark Mode.chromium.png|2| Changed|
|Table layout table.multiselect (mixed) - brand - High Contrast.chromium.png|113| Changed|
|Table layout table.multiselect (mixed) - brand - RTL.chromium.png|1| Changed|
|Table layout table.multiselect (mixed) - brand.chromium.png|1| Changed|
|Table layout table.multiselect (mixed) - neutral - High Contrast.chromium.png|77| Changed|
|Table layout table.multiselect (mixed) - neutral - RTL.chromium.png|4| Changed|
|Table layout table.multiselect (mixed) - neutral.chromium.png|4| Changed|
|Table layout table.multiselect - High Contrast.chromium.png|25| Changed|
|Table layout table.single select (checked) - High Contrast.chromium.png|25| Changed|
|Table layout table.single select (checked) - brand - High Contrast.chromium.png|69| Changed|
|Table layout table.single select (checked) - neutral - High Contrast.chromium.png|51| Changed|
|Table layout table.single select (checked) - neutral - RTL.chromium.png|2| Changed|
|Table layout table.single select (checked) - neutral.chromium.png|2| Changed|
|Table layout table.single select - High Contrast.chromium.png|25| Changed|
|Table layout table.size - medium - High Contrast.chromium.png|25| Changed|
|Avatar Converged.badgeMask - RTL.normal.chromium.png|0| Added|
|Avatar Converged.badgeMask.normal.chromium.png|0| Added|

</details>


<div id="vrtCommentfluentuiv9"/>

**Comment by behowell** [comment]
Merged this PR into #27527

**Pull request closed without merging**